docs/content/en/docs/how-tos/builders/_index.md

```diff
@@ -243,11 +243,15 @@ Skaffold will pass in the following additional environment variables for the fol
 | ------------- |-------------| -----|
 | Docker daemon environment variables     | Inform the custom builder of which docker daemon endpoint we are using. Allows custom build scripts to work with tools like Minikube. For Minikube, this is the output of `minikube docker-env`.| None. | 
 
+##### Cluster Builder
+| Environment Variable         | Description           | Expectation  |
+| ------------- |-------------| -----|
+| $KUBECONTEXT    | The expected kubecontext in which the image will be built.| None. | 
```

Comment:
This just occured to me: should we pass namespace as well?

Reply:
How about the secret names? 
Like...theoretically in cluster one should be able to generate a kaniko pod definition and run the build through that, right?

Reply:
good point, added the secrets, namespace and timeout.

I'm not sure what the best way to pass these [resource requirements](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/schema/latest/config.go#L246) is yet, but I'll tackle that in another PR.